https://github.com/arvidn/libtorrent/pull/6547

From 61ebb3317b1f8a63fcf4d018a21dac6818ea21cf Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Sun, 31 Oct 2021 09:16:46 +0000
Subject: [PATCH] Add missing include for get_ssl_category

error::get_ssl_category is defined in boost/asio/ssl/error.hpp, so
include it when we're building with SSL support.

Bug: https://bugs.gentoo.org/820836

---
 bindings/python/src/error_code.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/bindings/python/src/error_code.cpp b/bindings/python/src/error_code.cpp
index 2fc5f42..b947119 100644
--- a/bindings/python/src/error_code.cpp
+++ b/bindings/python/src/error_code.cpp
@@ -49,6 +49,7 @@ namespace boost
 
 #include <boost/asio/error.hpp>
 #if TORRENT_USE_SSL
+#include <boost/asio/ssl/error.hpp>
 #include <libtorrent/ssl.hpp>
 #endif
 #if TORRENT_USE_I2P
-- 
2.33.1

